sam/slack.py

```diff
@@ -59,6 +59,9 @@ async def handle_message(event: {str, Any}, say: AsyncSay):
     if event.get("subtype") == "message_deleted":
         logger.debug("Ignoring message_deleted event %s", event)
         return  # https://api.slack.com/events/message#hidden_subtypes
+    if event.get("subtype") == "message_changed":
```

Comment:
I think this can be an in list check and you can include the subtype in the message